.github/workflows/go.yml

```diff
@@ -15,23 +15,23 @@
 name: Go
 on:
   # Trigger the workflow on push or pull request,
-  # but only for the master branch
+  # but only for the 'next' branch
   push:
     branches:
-      - master
+      - next
```

Comment:
To prevent CI drift between the branches it may make sense to include all CI covered branches so the file can stay consistent between them.

Ex:

```diff
  branches:
    - master
+   - next
```

This shouldn't impact [execution of GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#:~:text=The%20following%20steps%20occur%20to%20trigger%20a%20workflow,%22Using%20environment%20variables.%22).

This way if new steps are introduced in `master` or `next` they can be introduced to the other more easily.